src/components/plates/PlateItem.vue

```diff
@@ -53,7 +53,7 @@ export default {
   methods: {
     getWellAt(position) {
       let well = this.wells.filter((well) => well.position == position)[0]
-      return well ? well : { position: position, materials: [] }
+      return well && well.materials !== undefined ? well : { position: position, materials: [] }
```

Comment:
Can you just do `well && well.materials`?
Also you can do `{ position, materials: [] }`
In javaScript if the property name equates to an existing variable it is inferred i.e. within an object position is the same as position: position.